js/googleMetadata.js

```diff
@@ -0,0 +1,253 @@
+
```

Comment:
Rename this file to `json-ld-validator.js` since it's closer to what it actually is doing.